### ISSUE TITLE
Add source map support to Karma test failures

### ISSUE THREAD

**Issue by stoltene2** [bazelbuild/rules_typescript#195]
While running the example angular-bazel-example, if you create a test failure you do not get a typescript stack trace. It would be great to utilize source maps.

```shell
angular-bazel-example (master)$ with_proxy bazel test //...
INFO: Analysed 19 targets (0 packages loaded).
INFO: Found 18 targets and 1 test target...
FAIL: //src/hello-world:test (see /private/var/tmp/_bazel_stoltene/2cc55f82f68f122d757ce425865745a9/execroot/angular_bazel_example/bazel-out/darwin-fastbuild/testlogs/src/hello-world/test/test.log)
INFO: From Testing //src/hello-world:test:
==================== Test output for //src/hello-world:test:
02 05 2018 13:27:28.590:INFO [karma]: Karma v1.7.1 server started at http://0.0.0.0:9876/
02 05 2018 13:27:28.592:INFO [launcher]: Launching browser ChromeHeadless with unlimited concurrency
02 05 2018 13:27:28.614:INFO [launcher]: Starting browser ChromeHeadless
02 05 2018 13:27:28.979:INFO [HeadlessChrome 64.0.3264 (Mac OS X 10.12.6)]: Connected on socket fP9nuUrB9l0zNdKBAAAA with id 80957237
HeadlessChrome 64.0.3264 (Mac OS X 10.12.6) BannerComponent (inline template) should display a different test title FAILED
        Expected 'Hello Test' to contain 'Test fails'.
            at UserContext.eval (http://concatjsangular_bazel_example/src/hello-world/hello-world.component.spec.js:47:36)
            at e.invoke (http://concatjsangular_bazel_example/node_modules/zone.js/dist/zone.min.js:1:22124)
            at e.onInvoke (http://concatjsangular_bazel_example/node_modules/zone.js/dist/proxy.min.js:1:2399)
            at e.invoke (http://concatjsangular_bazel_example/node_modules/zone.js/dist/zone.min.js:1:22064)
HeadlessChrome 64.0.3264 (Mac OS X 10.12.6): Executed 2 of 2 (1 FAILED) (0.068 secs / 0.065 secs)
================================================================================
INFO: Elapsed time: 5.849s, Critical Path: 5.09s
INFO: Build completed, 1 test FAILED, 3 total actions
//src/hello-world:test                                                   FAILED in 4.7s
  /private/var/tmp/_bazel_stoltene/2cc55f82f68f122d757ce425865745a9/execroot/angular_bazel_example/bazel-out/darwin-fastbuild/testlogs/src/hello-world/test/test.log

Executed 1 out of 1 test: 1 fails locally.

```

**Comment by alexeagle** [comment]
The file `bazel-bin/angular_bazel_example/src/hello-world/hello-world.component.spec.js` ought to have an inline sourcemap at the end if it was built by `ts_library`, so I wonder if there is something missing in Karma that doesn't use it to augment stack traces

**Comment by stoltene2** [comment]
@alexeagle I'll see if adding https://github.com/demerzel3/karma-sourcemap-loader solves the problem.

**Comment by alexeagle** [comment]
I think this was fixed by #196 ?

**Comment by stoltene2** [comment]
Yep
